docs/rules/space-before-blocks.md

```diff
@@ -49,7 +49,7 @@ class Foo{
 }
 ```
 
-The following patterns are not considered problems:
+Examples of **correct** code for this rule with the "always" option.
```

Comment:
You end with a colon on line 30 and a period here. I like the colon better, mind switching?